### PR TITLE
Add Humio

### PR DESCRIPTION
**Pull request by arcusfelis**
This PR describes how to configure structured logging with Humio.

Proposed changes include:
* also fixes formatting of one of OTP messages

### CODE REVIEW

**Comment by mongoose-im** [comment]
[8646.1](https://travis-ci.org/esl/MongooseIM/jobs/736184214) / Erlang 23.0.3 / small_tests / f944d7421aa07999a4c385504dde8044dc285eb1
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2913/8646/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2913/8646/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-10-15_21.39.00/lib.mongooseim.logs/run.2020-10-15_21.39.01/suite.log.html)

**Comment by mongoose-im** [comment]
[8647.1](https://travis-ci.org/esl/MongooseIM/jobs/736200345) / Erlang 23.0.3 / small_tests / ebd04c07b9b5cf0a7794de9ef3d2a7fb96242fc1
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2913/8647/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2913/8647/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-10-15_22.48.36/lib.mongooseim.logs/run.2020-10-15_22.48.36/suite.log.html)

**Review by janciesla8818 — COMMENTED** [review]
Looks good. Thanks for doing the tutorial and the fixes. Indeed I played a bit with the logs on humeo. Good stuff!

**Comment by mongoose-im** [comment]
[8656.1](https://travis-ci.org/esl/MongooseIM/jobs/737339148) / Erlang 23.0.3 / small_tests / ea8196c52d4a13dc3a5125b8c8882c3773d200a1
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2913/8656/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2913/8656/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-10-20_08.39.43/lib.mongooseim.logs/run.2020-10-20_08.39.44/suite.log.html)

**Review by NelsonVides — APPROVED** [review]

**Comment by mongoose-im** [comment]
[8668.1](https://travis-ci.org/esl/MongooseIM/jobs/737675091) / Erlang 23.0.3 / small_tests / 53e8e99e2228e5c56a1fc0a25f691f3996c81ee0
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2913/8668/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2913/8668/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-10-21_12.00.01/lib.mongooseim.logs/run.2020-10-21_12.00.01/suite.log.html)

**Review by chrzaszcz — CHANGES_REQUESTED** [review]
Thanks for introducing Humio. The integration works for me, but I think we broke logging along the way with the `lager` config. It looked suspicious from the beginning... could you propose a solution?

File: tools/test_runner/list_presets.erl [hunk 1]
```diff
@@ -12,8 +12,9 @@ main(_) ->
 -spec read_presets() -> [atom()].
 read_presets() ->
     {ok, Config} = file:consult("big_tests/test.config"),
-    Presets = proplists:get_value(ejabberd_presets, Config, []),
-    lists:map(fun({Name,_}) -> Name end, Presets).
+    Presets = proplists:get_value(presets, Config, []),
+    Toml = proplists:get_value(toml, Presets, []),
+    lists:map(fun({Name,_}) -> Name end, Toml).
```
Comment (chrzaszcz) [inline tools/test_runner/list_presets.erl:17]: Good catch :-)

**Comment by chrzaszcz** [comment]
Extract only the Humio-related changes and merge them.

**Comment by arcusfelis** [comment]
Replaced by #2952.

**Pull request closed without merging**